charts/ccpay-callback-function/Chart.yaml

```
@@ -25,6 +25,6 @@ dependencies:
     version: 2.0.1-beta
```

no harm updating this
```suggestion
    version: 2.3.4
```

---

DONE